packages/types/src/create/sanitize.ts

```diff
@@ -204,20 +205,26 @@ export function removePairOf (): Mapper {
 
 // remove the type traits
 export function removeTraits (): Mapper {
+  const from1 = /\s/g;
+  const from2 = /(T|Self)::/g;
+  const from3 = /<(T|Self)asTrait>::/g;
+  const from4 = /<Tas[a-z]+::Trait>::/g;
+  const from5 = /<LookupasStaticLookup>/g;
+  const from6 = /::Type/g;
```

Comment:
These could be moved to file-level, that is a small bump, i.e.doesn't need to be created each time in the function..

Reply:
They have pretty bad names that only make sense in a narrow scope.
Also this function is called just once, so will not make any difference in practice anyway.

Reply:
Cool, all good. At best it is a real micro-opt.